nixos/modules/services/networking/syncthing.nix

```diff
@@ -7,25 +7,20 @@ let
   opt = options.services.syncthing;
   defaultUser = "syncthing";
   defaultGroup = defaultUser;
+  settingsFormat = pkgs.formats.json { };
```

Comment:
@Xyz00777 can you elaborate what is this doing exactly? Could it be related to @azahi's issue?